torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -136,6 +136,14 @@ Node* addDummyClone(
       orig_data->type()->kind() == TypeKind::BoolType) {
     auto* noneNode = graph->create(prim::Constant);
     noneNode->output()->setType(NoneType::get());
+    // For scripting mode, aten::clone requires input to be a TensorType
+    // Hence if we encounter an IntType or loatType, we set the input
+    // to the appropriate TensorType
+    if (orig_data->type()->kind() == TypeKind::IntType) {
+      orig_data->setType(TensorType::fromNumberType(*IntType::get()));
+    } else if (orig_data->type()->kind() == TypeKind::FloatType) {
```

Comment:
Do we have to care about int8, float16, float64 or `IntType::get()`/`FloatType::get()` already handle these cases?